docs/sources/helm-charts/mimir-distributed/release-notes/v5.5.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Grafana Mimir Helm chart version 5.5 release notes"
+menuTitle: "V5.5 release notes"
+description: "Release notes for Grafana Mimir Helm chart version 5.5"
+weight: 300
+---
+
+# Grafana Mimir Helm chart version 5.5 release notes
+
+Grafana Labs is excited to announce version 5.5 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.14 or Grafana Enterprise Metrics (GEM) v2.14. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.
+
+The highlights that follow include the top features, enhancements, and bug fixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+
+## Important changes
+
+- The ingester is configured with `GOMAXPROCS` to help with the overhead from Go scheduler, observed on hosts with lots of CPU cores.
+- The GEM query-frontend logs the query author's access policy name and token name in the query statistics logs. This provides a better visibility of who (which token) is running a potentially expensive query.
```

Comment:
doesn't this belong to the GEM changelog? or was it a change to the chart?

Reply:
We have this in GEM changelog also. This one is about making the feature configured and enabled by default in the chart.